src/load_tests/check.h

```diff
@@ -0,0 +1,11 @@
+#ifndef LT_CHECK_H
+#define LT_CHECK_H
+
+#include <string>
+#include <boost/filesystem.hpp>
+
+const std::string CHECK_FOR_UPDATES = std::string{"check_for_update"};
```

Comment:
No need to make copies here.
Could be 
`const std::string CHECK_FOR_UPDATES{"check_for_update"};`